packages/jellyfish-transaction-builder/src/txn/txn_builder_loans.ts

```diff
@@ -173,4 +174,19 @@ export class TxnBuilderLoans extends P2WPKHTxnBuilder {
       changeScript
     )
   }
+
+  /**
+   * Close a vault.
+   *
+   * @param {CloseVault} closeVault txn to create
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   * @returns {Promise<TransactionSegWit>}
+   */
+
```

Comment:
```suggestion
```

Reply:
looks okay? one line break

Reply:
yea.. just extra break line

Reply:
Ok. I am going to fix this in another PR.

Reply:
Hmm, this seems correct to me. A break line should be added after the comment section.
Anyway, I have group all the functions name ended with `Vault` together

<img width="910" alt="Screenshot 2021-11-09 at 12 18 11 AM" src="https://user-images.githubusercontent.com/7101806/140778716-a6bf32be-a7cf-4dba-a00f-6ba1519dc1e3.png">

Reply:
Fixed in updateVault DFTX PR